packages/tools/nodeEditor/src/blockTools.ts

```diff
@@ -355,6 +355,11 @@ export class BlockTools {
                 triangleWaveBlock.kind = WaveBlockKind.Triangle;
                 return triangleWaveBlock;
             }
+            case "SetBlock": {
+                const cosBlock = new TrigonometryBlock("Set");

```

Comment:
nit: `setBlock` would be better than `cosBlock`

Reply:
Okay, I has fixed it.

Reply:
next week, I will create a to Tool  to convert GLSL code to NME, so we need the "SetBlock".